### PR TITLE
fix: edge register bundle `IsClusterDialerClientRegistered` force use client type

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
edge register bundle `IsClusterDialerClientRegistered` force use client type

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |   Fix the bug that edge register bundle `IsClusterDialerClientRegistered` force use client type           |
| 🇨🇳 中文    |     强制dialer的一些bundle使用类型参数         |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/pipeline/providers/edgepipeline_register/interface_test.go [hunk 1]
```diff
@@ -238,3 +238,42 @@ func TestIsEdge(t *testing.T) {
 	}
 	assert.Equal(t, true, p.IsEdge())
 }
+
+func TestShouldDispatchToEdge(t *testing.T) {
```
Comment (sfwn) [inline modules/pipeline/providers/edgepipeline_register/interface_test.go:242]: it seems to be an unrelated unit test.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/providers/edgepipeline_register/interface_test.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/providers/edgepipeline_register/interface_test.go:242]: yes, `ShouldDispatchToEdge` only use `IsClusterDialerClientRegistered` to judge is registed

**Review by sfwn — APPROVED** [review]
